pretty_midi/pretty_midi.py

```diff
@@ -49,19 +47,24 @@ def __init__(self, midi_file=None, resolution=220, initial_tempo=120.):
         """
         if midi_file is not None:
             # Load in the MIDI data using the midi module
-            midi_data = midi.read_midifile(midi_file)
+            midi_data = mido.MidiFile(midi_file)
```

Comment:
In order to continue support loading midi files from file objects, this should be something like:

```
            if type(midi_file) in (str, unicode):
              midi_data = mido.MidiFile(filename=midi_file)
            else:
              midi_data = mido.MidiFile(file=midi_file)
```

Which is similar to what the previous midi package did:
https://github.com/vishnubob/python-midi/blob/master/src/fileio.py#L158